src/client/utils/common.js

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line import/prefer-default-export
+export const cloneDeep = (el) => JSON.parse(JSON.stringify(el));
```

Comment:
Can't use anything in `lodash`? `JSON.parse` and `JSON.stringify` can throw errors.